pkg/server/container_create.go

```diff
@@ -193,7 +193,7 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	meta.StopSignal = image.ImageSpec.Config.StopSignal
 
 	// Get container log path.
-	if config.GetLogPath() != "" {
+	if config.GetLogPath() != "" && sandboxConfig.GetLogDirectory() != "" {
```

Comment:
can we update the comment line above and add a logrus.Debugf to aide in debugging the invalid config issue..

Reply:
Done. Let me know if you like the new comment and log message.